### ISSUE TITLE
Bug: Handle connection reset by peer

### ISSUE THREAD

**Issue by BenB196** [BenB196/crashplan-ffs-puller#34]
The following error can be thrown and should be handled.

```
panic: Post https://forensicsearch-default.prod.ffs.us2.code42.com/forensic-search/queryservice/api/v1/fileevent/export: read tcp 10.111.1.8:51048->3.87.164.248:443: read: connection reset by peer

goroutine 422582 [running]:
github.com/BenB196/crashplan-ffs-puller/ffsEvent.queryFetcher(0xc000024da0, 0xa, 0xc000022680, 0x17, 0xc000024db0, 0xf, 0xc000024daa, 0x2, 0xc000024dac, 0x3, ...)
        /app/ffsEvent/ffsEvent.go:249 +0x2c05
created by github.com/BenB196/crashplan-ffs-puller/ffsEvent.FFSQuery.func5
        /app/ffsEvent/ffsEvent.go:185 +0x1e9
panic: read tcp 10.111.1.8:51048->3.87.164.248:443: read: connection reset by peer

goroutine 422473 [running]:
github.com/BenB196/crashplan-ffs-puller/ffsEvent.queryFetcher(0xc000024da0, 0xa, 0xc000022680, 0x17, 0xc000024db0, 0xf, 0xc000024daa, 0x2, 0xc000024dac, 0x3, ...)
        /app/ffsEvent/ffsEvent.go:249 +0x2c05
created by github.com/BenB196/crashplan-ffs-puller/ffsEvent.FFSQuery.func5
        /app/ffsEvent/ffsEvent.go:185 +0x1e9
```

**Comment by BenB196** [comment]
This is another example of the error happening:

```
panic: read tcp 10.111.1.8:52760->34.225.168.230:443: read: connection reset by peer

goroutine 638911 [running]:
github.com/BenB196/crashplan-ffs-puller/ffsEvent.queryFetcher(0xc00011cd90, 0xa, 0xc00011e360, 0x17, 0xc00011cda0, 0xf, 0xc00011cd9a, 0x2, 0xc00011cd9c, 0x3, ...)
        /app/ffsEvent/ffsEvent.go:249 +0x2c05
created by github.com/BenB196/crashplan-ffs-puller/ffsEvent.FFSQuery.func5
        /app/ffsEvent/ffsEvent.go:185 +0x1e9
```

**Comment by BenB196** [comment]
Fixed in release [v0.1.2](https://github.com/BenB196/crashplan-ffs-puller/releases/tag/v0.1.2).
